pkg/gpu/nvidia/health_check/health_checker.go

```diff
@@ -159,6 +166,56 @@ func (hc *GPUHealthChecker) addMigEnabledDevice(deviceName string, device *nvml.
 	return nil
 }
 
+type callDevice interface {
+	parseMigDeviceUUID(UUID string) (string, uint, uint, error)
+}
+type GPUDevice struct{}
+
+func (gd *GPUDevice) parseMigDeviceUUID(UUID string) (string, uint, uint, error) {
+	return nvml.ParseMigDeviceUUID(UUID)
+}
+
+func (hc *GPUHealthChecker) catchError(e nvml.Event, cd callDevice) {
+	if e.Etype != nvml.XidCriticalError {
+		return
+	}
+	// Only marking device unhealthy on Double Bit ECC Error
+	// See https://docs.nvidia.com/deploy/xid-errors/index.html#topic_4
+	if _, ok := hc.healthCriticalXid[e.Edata]; !ok {
+		return
```

Comment:
nit: can we add a log here to print out the e.Edata, so that user can use this log to check which XID they skipped, and help them to better customize this XID in the future. Also, helpful for debugging.